### PR TITLE
Mantis blade (new traitor item)

### PR DESCRIPTION
**Pull request by enkidienne**
# General Documentation

Adds a new traitor item acquirable through any uplink for an expensive 16 TC. Your 16 TC gets you a fancy briefcase. Inside of it is 2 autodoc, 1 for each arm. Once used they will implant in your body a set of 2 retractable blades. These implants are stealthy and won't show on scans. The blades do 20 damage. Each attack, both blades will attack at once, dealing a total of 40 damage. The mantis blades are very noticeable once out so be careful. I have included a Nanostrasen version, but it is currently unobtainable. 

### Intent of your Pull Request

moar fun

### Why is this change good for the game?

moar fun

# Wiki Documentation

The Mantis blades do 20 damage each and both attack at once.
The Nanostrasen version does 18 damage and both attack at once.

### Briefly describe your PR and the impacts of it, in layman's terms.
 
New blades, traitor item. Blades do big damage when paired.

### What should players be aware of when it comes to the changes your PR is implementing?

Beware of the people whose arms open to reveal dark and deadly blades. 

### What general grouping does this PR fall under? 

Syndicate

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

no

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

Syndi blade: 20 damages
Paired syndi blades: 40 damages
NT blade: 18 damages
Paired NT blades: 36 damages

# Changelog

add: mantis blades

:cl:  

rscadd: mantis blade

/:cl:

mantis

unknown

on the right is the NT version and the left is syndie

ALL CREDIT GOES TO TG, I MERELY PORTED IT AND MAKE IT VIABLE. 
original author: EdgeLordExe
https://github.com/tgstation/tgstation/pull/55936

### CODE REVIEW

**Comment by Tipy1802** [comment]
If you want to make the NT blades obtainable perhaps lock them behind illegal tech and have them researched at the sec lathe

**Comment by SomeguyManperson** [comment]
link the original pr in the description if it's a port

**Comment by enkidienne** [comment]
> link the original pr in the description if it's a port

done

File: code/game/objects/items/mantis.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/obj/item/mantis/blade
+	name = "mantis blade"
+	desc = "Powerful inbuilt blade, hidden just beneath the skin. Singular brain signals directly link to this bad boy, allowing it to spring into action in just seconds."
```
Comment (Moggram) [inline code/game/objects/items/mantis.dm:3]: "A blade designed to be hidden just beneath the skin. The brain is directly link to this bad boy, allowing it to spring into action."

You don't  have to use it, but I needed to reread that description.

File: code/modules/surgery/organs/augments_arms.dm [hunk 2]
```diff
@@ -233,3 +233,13 @@
 	name = "surgical toolset implant"
 	desc = "A set of surgical tools hidden behind a concealed panel on the user's arm."
 	contents = newlist(/obj/item/retractor/augment, /obj/item/hemostat/augment, /obj/item/cautery/augment, /obj/item/surgicaldrill/augment, /obj/item/scalpel/augment, /obj/item/circular_saw/augment, /obj/item/surgical_drapes)
+
+/obj/item/organ/cyberimp/arm/syndie_mantis
+	name = "G.O.R.L.E.X. mantis blade implants"
+	desc = "Modernized mantis blade designed coined by Tiger operatives, much sharper blade with energy actuators makes it a much deadlier weapon."
```
Comment (Moggram) [inline code/modules/surgery/organs/augments_arms.dm:239]: "Modernized mantis blade designed and coined by Tiger operatives. Energy actuators makes the blade a much deadlier weapon."

as before, you don't have to use it.

**Comment by Marmio64** [comment]
maybe lower damage and price so that you can reasonably afford another tool. Most syndi items are designed to be cheap enough to allow you to purchase something else too so all of your antag abilities don't pivot on a single item, which is especially bad when its just a rather mundane weapon like this is rather than being a versatile martial art with it's own gimmick and multiple moves like sleeping carp

**Comment by enkidienne** [comment]
> maybe lower damage and price so that you can reasonably afford another tool. Most syndi items are designed to be cheap enough to allow you to purchase something else too so all of your antag abilities don't pivot on a single item, which is especially bad when its just a rather mundane weapon like this is rather than being a versatile martial art with it's own gimmick and multiple moves like sleeping carp

the price is currently at 16 TC, AKA the same price as a double desword. The Mantis blades are meant as an alternative to deswords as they have about 8 more damage, no recongnizable shshhhh sounds, yet they cant block nor deflect anything. AKA, a more offense-orientated weapon that while more dangerous than the desword, is also less forgiving.

File: code/modules/surgery/organs/augments_arms.dm (hunk 2 above)
Comment (enkidienne) [reply code/modules/surgery/organs/augments_arms.dm:239]: It seems that this is the limit of my English skills? I don't really get what you mean...?

File: code/game/objects/items/mantis.dm (hunk 1 above)
Comment (enkidienne) [reply code/game/objects/items/mantis.dm:3]: It seems that this is the limit of my English skills? I don't really get what you mean...?

Comment (Moggram) [reply code/game/objects/items/mantis.dm:3]: I am giving you a new description to use. You do not have to use it, but read both comments I have made and see if you like them. I hope this clears up any miss understandings.

**Review by nesisknesis — APPROVED** [review]

**Comment by SomeguyManperson** [comment]
fix merge conflicts

**Review by SomeguyManperson — APPROVED** [review]

**Comment by boodaliboo** [comment]
fix conflicts

**Comment by enkidienne** [comment]
I've tried fixing them conflicts for some time now, and ain't nothing working. Multiple people tried to help but nothing seems to work so at this point I think this is way beyond what I'm capable of. if you want to take a shot at it, feel free, but I don't think ill be able to fix this PR without a major breakthrough.

**Comment by adamsong** [comment]
I'm not sure what you have tried at this point but what I would suggest is on your local branch, back up the three conflicting icon files, merge, use the master icon files, and copy the icons over. The surgeon should be a fairly easy merge

**Comment by enkidienne** [comment]
> I'm not sure what you have tried at this point but what I would suggest is on your local branch, back up the three conflicting icon files, merge, use the master icon files, and copy the icons over. The surgeon should be a fairly easy merge

Tried, didnt work.

**Comment by JamieD1** [comment]
40 damage for hideable arms feels a bit op no? @SomeguyManperson Your view?
